src/ric/d_18C40.c

```diff
@@ -0,0 +1,330 @@
+    0x02, 0x02, 0x02, 0x00, 0x33, 0x00, 0x00, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x06, 0x7F, 0xFF, 0xFF,
+
+    0x01, 0x01, 0x04, 0x00, 0x73, 0x00, 0x03, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x0C, 0xC0, 0x60, 0x00,
+
+    0x01, 0x01, 0x04, 0x00, 0x33, 0x00, 0x03, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x0C, 0x7F, 0x00, 0x00,
+
+    0x03, 0x03, 0x02, 0x00, 0x02, 0x00, 0x04, 0x00, 0x00, 0x00, 0x80, 0x0C,
+    0x08, 0x1F, 0x1F, 0x7F,
+
+    0x01, 0x01, 0x04, 0x00, 0x33, 0x00, 0x06, 0x00, 0x00, 0x00, 0x80, 0x0C,
+    0x14, 0x7F, 0x7F, 0xC0,
+
+    0x01, 0x01, 0xFE, 0xFF, 0x33, 0x00, 0x07, 0x00, 0x00, 0x00, 0x80, 0x0C,
+    0x06, 0xC0, 0xC0, 0xC0,
+
+    0x02, 0x02, 0x02, 0x00, 0x7B, 0x00, 0x08, 0x00, 0x00, 0x00, 0x80, 0x08,
+    0x10, 0x7F, 0x7F, 0x7F,
+
+    0x01, 0x01, 0x02, 0x00, 0x33, 0x00, 0x09, 0x00, 0x00, 0x00, 0x80, 0x08};
+
+unkStr_8011E4BC* D_80154DA0[] = {
+    &D_80154CF4[0xC],  &D_80154CF4[0x1C], &D_80154CF4[0x2C], &D_80154CF4[0x3C],
+    &D_80154CF4[0x4C], &D_80154CF4[0x5C], &D_80154CF4[0x6C], &D_80154CF4[0x7C],
+    &D_80154CF4[0x8C], &D_80154CF4[0x9C]};
+
+AnimationFrame D_80154DC8[] = {
+    0x0002, 0x0001, 0x0002, 0x0002, 0x0002, 0x0003, 0x0002, 0x0004,
+    0x0002, 0x0005, 0x0002, 0x0004, 0x0002, 0x0003, 0x0002, 0x0004,
+    0x0002, 0x0003, 0x0002, 0x0004, 0x0002, 0x0005, 0x0001, 0x0006,
+    0x0001, 0x0007, 0x0001, 0x0008, 0xFFFF, 0x0000,
+};
+
+AnimationFrame D_80154E04[] = {
+    0x0001, 0x0009, 0x0002, 0x000A, 0x0002, 0x000B, 0x0002, 0x000C, 0x0002,
+    0x000D, 0x0002, 0x000E, 0x0002, 0x000F, 0x0002, 0x0010, 0x0002, 0x0011,
+    0x0002, 0x0012, 0x0003, 0x0013, 0x0004, 0x0014, 0xFFFF, 0x0000,
+};
+
+AnimationFrame D_80154E38[] = {
+    0x0002, 0x0001, 0x0002, 0x0002, 0x0002, 0x0003, 0x0002, 0x0004, 0x0002,
+    0x0005, 0x0002, 0x0006, 0x0002, 0x0007, 0x0002, 0x0008, 0xFFFF, 0x0000,
+};
+
+unkStr80154E5C D_80154E5C[] = {
+    {
+        0xFFC0,
+        0x0000,
+        0x00028000,
+        0x00000000,
+        0x0060,
+        0x001B,
+        0x0118,
+        0x80,
+        0x00,
+    },
+    {
+        0x0040,
+        0x0000,
+        0xFFFD8000,
+        0x00000000,
+        0x0048,
+        0x001B,
+        0x0119,
+        0x00,
+        0x80,
+    },
+    {
+        0x0000,
+        0xFFC0,
+        0x00000000,
+        0x00028000,
+        0x0030,
+        0x0019,
+        0x011A,
+        0x00,
+        0x00,
+    },
+    {
+        0x0000,
+        0x0040,
+        0x00000000,
+        0xFFFD8000,
+        0x0018,
+        0x0019,
+        0x011B,
+        0x80,
+        0x00,
+    },
+};
+
+s16 D_80154EAC[] = {
+    0x016E,
+    0x0161,
+    0x0160,
+    0x0162,
+};
+
+s32 D_80154EB4[] = {
+    0x0001AAAA,
+    0xFFFE5556,
+    0x00005555,
+    0xFFFF2AAB,
+};
+
+s32 D_80154EC4[] = {
+    0xFFFE0000,
+    0xFFFE5556,
+    0xFFFD0000,
+    0xFFFDAAAB,
+};
+
+s32 D_80154ED4[] = {
+    0x00010004, 0x00020004, 0x00030004, 0x00040004, 0x00050004,
+    0x00060004, 0x00070004, 0x00080004, 0x00000000,
+};
+
+s32 D_80154EF8[] = {
```

Comment:
```suggestion
s32 D_80154EF8[] = {
```
This is an array of `AnimationFrame`